modules/Hoot/config/domMetadata.js

```diff
@@ -397,6 +398,12 @@ export function exportDataForm( zipOutput ) {
         d3.select( `#${exportFgdbId}_container` )
             .classed( 'hidden', !showFgdbTemplate );
 
+        const showThematic = d3.select( `#${exportFormatId}` ).property( 'value' ).indexOf('OpenStreetMap') === -1
+                        && d3.select( `#${exportComboId}` ).property( 'value' ).indexOf('OSM') === -1;
```

Comment:
Show option to suppress thematic grouping of layers only when both format and schema are not osm.